### ISSUE TITLE
keyboard problem / sent text not shown in conversation

### ISSUE THREAD

**Issue by peterpanondrugs** [wireapp/wire-android#595]
Had it only once. But I have a lot of problems with the keyboard. ?!

- go to chat
- type smth
- send it
- wondering why the written text is not shown.
- start wire again or maybe the trick with opening another chat. Helped.
See log
[wire_debug_report.txt](https://github.com/wireapp/wire-android/files/768012/wire_debug_report.txt)

**Comment by xuanzhaopeng** [comment]
Hey @peterpanondrugs , in fact I suffer the same problem only one time.  I'd like to know which version of keyboard you use? I will try to reproduce it in Monday.

**Comment by peterpanondrugs** [comment]
![tmp_6335-screenshot_20170212-100128662549015](https://cloud.githubusercontent.com/assets/8776733/22862903/2ed90880-f140-11e6-9f9c-a854bde87f09.png)

**Comment by peterpanondrugs** [comment]
[wire_debug_report.txt](https://github.com/wireapp/wire-android/files/833762/wire_debug_report.txt)

**Comment by nichii124** [comment]
My friend is having a similar issue. I'm sending her messages on my Linux computer and she checks them on her Android phone, but there's no messages to read. I'm able to read her messages from her Android phone on my computer just fine though.

**Comment by Gardosen** [comment]
as there was no response over the past nearly 3 years, i am closing this ticket as "resolved by client progress".

if anyone has still such an issue, he is free to reopen another issue ticket to let us know with more details based on the current client and state.

Kind regards
Marco
